thipster/parser/dsl_parser/lexer_position.py

```diff
@@ -1,12 +1,15 @@
+"""Module to represent the state and position of the lexer."""
 from thipster.engine.parsed_file import Position
 
 
 class LexerPosition():
+    """Class to represent the state and position of the DSL lexer."""
+
     def __init__(
         self,
         filename: str = '',
     ) -> None:
-        """Class to represent the state and position of the lexer
+        """Class to represent the state and position of the lexer.
```

Comment:
^

Here you use the class docstring whereas in previous init methods you use `Initialize a XYZ object.` syntax maybe we should harmonize it ?
fyi other packages seem to use something similar to the class doctring